sbin/spark-daemon.sh

```diff
@@ -129,8 +129,9 @@ case $option in
     mkdir -p "$SPARK_PID_DIR"
 
     if [ -f $pid ]; then
-      if kill -0 `cat $pid` > /dev/null 2>&1; then
-        echo $command running as process `cat $pid`.  Stop it first.
+      TARGET_ID=`cat $pid`
```

Comment:
Bash style: Use `$(...)` instead of backticks, which are discouraged.

Also, quote the output of the subshell, as well as the variable argument to `cat`:

```
TARGET_ID="$(cat "$pid")"
```